src/components/PomodoroButtons/PomodoroButton/PomodoroButton.module.scss

```diff
@@ -0,0 +1,40 @@
+
+.button {
+  width: 6.56rem;
+  height: 3rem;
+  font-family: 'Kumbh Sans', sans-serif;
+  font-size: 0.75rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: normal;
+  border-radius: 1.66rem;
+  color: #D7E0FF;
+  background: #161932;
+  opacity: 0.4;
+  &:focus {
+    outline: none;
+  }
```

Comment:
I would be useful to add also styles for focus-visible to have some kind of visual indication of where user currently is on the page when navigating via keyboard

Reply:
Done